x-pack/solutions/security/plugins/security_solution/public/flyout/ui_tests/tests/alert_details_url_sync.spec.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { expect, test } from '@kbn/scout-security';
+import { CUSTOM_QUERY_RULE } from '@kbn/scout-security/src/playwright/constants/detection_rules';
+
+const RIGHT = 'right';
+
+test.describe('Expandable flyout state sync', { tag: ['@ess', '@svlSecurity '] }, () => {
+  test.beforeEach(async ({ browserAuth, detectionRuleApi }) => {
+    await detectionRuleApi.deleteAll();
+    await detectionRuleApi.createCustomQueryRule();
```

Comment:
While looking into Fleet API example I did

```
  test.afterEach(async ({ fleetApi }) => {
    await fleetApi.integration.delete(integrationName);
  });
```

I wonder if we should move unify under `kbnApis` or smth similar so you can do something like

```
  test.beforeEach(async ({ browserAuth, kbnApi }) => {
    await kbnApis.detectionRule.deleteAll();
    await kbnApis.detectionRule.createCustomQueryRule();

    await kbnApis.fleet.integration.delete(integrationName);
    await kbnApis.alerting.anotherCoolMethod();
```

It should help us avoid importing each API fixture + everyone will know that `kbnAPIs` wraps different Kibana APIs?

WDYT?

Reply:
And move this to scout instead of having it inside scout-security?

Reply:
no, it should stay in @kbn/scout-security and extends the `kbnApi` fixture imported from @kbn/scout. I will raise a PR in @kbn/scout